kustomize/tests/spanner-with-all-components/kustomization.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- ../../base
+components:
+- ../../components/cymbal-branding
+- ../../components/google-cloud-operations
+- ../../components/network-policies
+- ../../components/service-accounts
```

Comment:
Thought:
Again, this approach is cleaner!
We will just have to remember to add any _new_ Kustomize components to each `kustomize/test/`.